packages/cli-doctor/src/tools/runBundleInstall.ts

```diff
@@ -0,0 +1,21 @@
+import execa from 'execa';
+import {logger} from '@react-native-community/cli-tools';
+
+import {Loader} from '../types';
+
+async function runBundleInstall(loader: Loader) {
+  try {
+    loader.start('Installing Bundler');
```

Comment:
Why this line does not appear in the successful run posted in the Test Plan? 🤔

Reply:
> Why this line does not appear in the successful run posted in the Test Plan? 🤔

Looks like I need to make a separate `logger.info` call, will fix that